spec/filters/aggregate_spec.rb

```diff
@@ -0,0 +1,203 @@
+# encoding: utf-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/filters/aggregate"
+
+describe LogStash::Filters::Aggregate do
+	def event(data = {})
```

Comment:
this helper methods are better placed in a spec helper file, then you can reference it here with ```require_relative .... ```

if you wan to be a hardcode rspec hacker, you can also look at ways to extend rspec with custom matchers (https://www.relishapp.com/rspec/rspec-expectations/v/2-4/docs/custom-matchers/define-matcher), but this is actually just if you want to learn it :+1: 

so helper methods, better places in a helper file.